specs/interop/messaging.md

```diff
@@ -162,6 +165,34 @@ The expiry invariant invalidates inclusion of any executing message with
 - `executing_block` is the block where the executing message was included in.
 - `EXPIRY_TIME = 180 * 24 * 60 * 60 = 15552000` seconds, i.e. 180 days.
 
+## Special Messages
+
+### `logIndex` wildcard
+
+[wildcard]: #logindex-wildcard
+
+The author of an executing-transaction may defer the specification of an exact `logIndex` to the block-builder.
+Deferring the `logIndex` enables the executing side to request a message to be initiated.
+The `logIndex` value of `0x01000000000000000000000000000000000000000000000000000000000000` functions as wildcard.
```

Comment:
This value is only 31 bytes long which I suspect is a typo?

Also how was this value chosen? It doesn't seem very unique, perhaps using `bytes32(uint256(keccak256("interop.logIndexWildcard")) - 1)` is safer (this is the same ERC-1967 slot computation pattern used in contracts). Or even just converting a string like "interop log index wildcard" to hex and padding it